radixdlt/src/integration/java/com/radixdlt/consensus/simulation/BFTSimulatedTest.java

```diff
@@ -142,7 +144,11 @@ public Builder checkNoSyncExceptions(String checkName) {
 		public Builder checkAllProposalsHaveDirectParents(String checkName) {
 			this.checksBuilder.put(checkName, new AllProposalsHaveDirectParentsCheck());
 			return this;
+		}
 
+		public Builder checkNoneCommitted(String checkName) {
+			this.checksBuilder.put(checkName, new NoneCommittedCheck());
```

Comment:
what is a "check name"? Shoulds "checks" be renamed "milestone"? Or "threshold" or is it more like a validation/prerequisites/fulfillment? This would allow for variable names such as:

`nameOfFulfilledRequirement` or similar, which IMO is a better variable name than `checkName`?

Reply:
perhaps `invariant` is the best but would rather do this in a different PR.